bears/js/JSONFormatBear.py

```diff
@@ -90,9 +90,15 @@ def run(self, filename, file,
         diff = Diff.from_string_arrays(file, corrected)
 
         if len(diff) > 0:
+            # Appropriate message is displayed to the user based on
```

Comment:
Please change the comment describing the parameter to this format:
"""
:param format_message: (a brief description about the parameter)
"""

Reply:
@li-boxuan Is the comment format wrong?

Reply:
Yes, I thought so. Will remove it in next commit.